package.json

```
@@ -174,7 +174,8 @@
           "sourcekit-lsp.inlayHints.enabled": {
             "type": "boolean",
             "default": true,
-            "description": "Render inlay type annotations in the editor. Inlay hints require Swift 5.6 or later.",
+            "description": "Render inlay type annotations in the editor. Inlay hints require Swift 5.6.",
+            "deprecationMessage": "SourceKit-LSP's custom inlay hints are deprecated as Swift 5.7 and newer use LSP-native inlay hints. These can be enabled using the setting 'editor.inlayHints.enabled'.",
```

If you use the markdown deprecation you can add a link to the setting. Also lets keep the message simple.
```suggestion
            "markdownDeprecationMessage": "**Deprecated**: Please use `#editor.inlayHints.enabled#` instead.",
```